src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -3189,7 +3189,7 @@ def test_network_watcher_flow_log(self, resource_group, resource_group_location,
         ])
 
     @ResourceGroupPreparer(name_prefix='cli_test_nw_flow_log2', location='canadaeast')
-    @StorageAccountPreparer(name_prefix='clitestnw', location='canadaeast')
+    @StorageAccountPreparer(name_prefix='clitestnw', location='canadaeast', kind='StorageV2')
```

Comment:
Out of dated way to configure flow-log of watcher. There must be changes on service to restrict only V2 to configure a flow-log instead the original V1.